src/components/TextInput/TextInput.module.scss

```diff
@@ -1,16 +1,16 @@
 .textInput {
-  background-color: var(--form-control-background-color);
-  border: 1px solid var(--form-control-border-color);
   box-sizing: border-box;
-  box-shadow: 0px 3px 0px rgba(0, 0, 0, 0.05);
+  transition-duration: 300ms;
```

Comment:
I know, these look like crazy changes, but, it's just stylelint auto-fixing our files, largely to re-organize properties based on the Concentric CSS pattern.